### PR TITLE
feat(render): add audio rendering callback, return audio data that ha…

### PR DESCRIPTION
**Pull request by kongjianneu**
…s processed

### CODE REVIEW

File: framework/render/audio/IAudioRender.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #include <base/media/IAFPacket.h>
 #include <utils/AFMediaType.h>
+#include <native_cicada_player_def.h>
```
Comment (pingkai) [inline framework/render/audio/IAudioRender.h]: framework should not dependent mediaplayer

File: framework/render/audio/IAudioRender.h [hunk 2]
```diff
@@ -164,9 +165,17 @@ namespace Cicada{
             mListener = listener;
         }
 
+        virtual void setRenderingCb(onRenderFrame cb, void *userData)
+        {
+            mRenderingCb = cb;
+            mRenderingCbUserData = userData;
+        }
+
```
Comment (pingkai) [inline framework/render/audio/IAudioRender.h:175]: should not set the rendering callback to filter

File: mediaPlayer/SuperMediaPlayer.cpp [hunk 3]
```diff
@@ -4209,6 +4210,16 @@ void SuperMediaPlayer::SetOnRenderCallBack(onRenderFrame cb, void *userData)
     mFrameCb = cb;
     mFrameCbUserData = userData;
 }
+
+void SuperMediaPlayer::SetAudioRenderingCallBack(onRenderFrame cb, void *userData)
+{
+    mAudioRenderingCb = cb;
+    mAudioRenderingCbUserData = userData;
+    if (mAudioRender) {
```
Comment (pingkai) [inline mediaPlayer/SuperMediaPlayer.cpp]: Can‘t update the callback here，it's not thread safe，SetAudioRenderingCallBack can only effect when set before calling prepare

**Review by pingkai — CHANGES_REQUESTED** [review]

File: framework/render/audio/IAudioRender.h (hunk 2 above)
Comment (kongjianneu) [reply framework/render/audio/IAudioRender.h:175]: set the callback in IAudioRender

**Review by pingkai — APPROVED** [review]

**Pull request merged**